lib/Alchemy/Phrasea/Core/Event/Subscriber/DataboxSubscriber.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of Phraseanet
+ *
+ * (c) 2005-2014 Alchemy
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Alchemy\Phrasea\Core\Event\Subscriber;
+
+use Alchemy\Phrasea\Core\Event\DataboxEvent\DataboxIndexEvent;
+use Alchemy\Phrasea\Core\Event\DataboxEvent\DeleteStatusEvent;
+use Alchemy\Phrasea\Core\Event\DataboxEvent\UpdateStatusEvent;
+use Alchemy\Phrasea\Core\Event\DataboxEvent\UpdateStructureFieldEvent;
+use Alchemy\Phrasea\Core\PhraseaEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class DataboxSubscriber implements EventSubscriberInterface
+{
+    public static function getSubscribedEvents()
+    {
+        return [
+            PhraseaEvents::DATABOX_UPDATE_FIELD => 'onUpdateField',
+            PhraseaEvents::DATABOX_DELETE_FIELD => 'onDeleteField',
+            PhraseaEvents::DATABOX_UPDATE_STATUS => 'onUpdateStatus',
+            PhraseaEvents::DATABOX_DELETE_STATUS => 'onDeleteStatus'
+        ];
+    }
+
+    public function onUpdateField(UpdateStructureFieldEvent $event)
+    {
+        $databox = $event->getDatabox();
+        $field = $event->getField();
+        $data = $event->getData();
+
+        if ((bool) $field->get_thumbtitle() !== (bool) $data['thumbtitle']) {
```

Comment:
It just checks if user has change the property 'title' of a field, if yes we must reindex the data box because it changes the title of all records